utils/monitoring_tools/hardware_monitoring/README.md

```diff
@@ -0,0 +1,158 @@
+# Neptune Hardware Monitoring
```

Comment:
Forgot to mention: In general, we use sentence case in headings (only capitalize words if they should be capitalized anywhere)